app/components/collections/show/settings_header_component.html.erb

```diff
@@ -12,6 +12,9 @@
     </div>
   </div>
   <span class="header-text"><%= name %></span><%= render Collections::DraftComponent.new(collection_version: collection_version) %>
+  <% if collection.head&.decommissioned? %>
+    <span class="state">Decommissioned</span>
+  <% end %>
```

Comment:
I wonder too if this could be rolled into a renamed `Collections::DraftComponent`, though it could be this is used in contexts where we don't want the decommissioned label?

Reply:
It is a component for works, but that seemed like too much overhead for something this simple.  Also, there's a similar if statement a few lines up.

Reply:
oh my - DraftComponent is ridiculously simple as well.

Reply:
I decided to keep this as is because the DraftComponent is used elsewhere:

![image](https://user-images.githubusercontent.com/96775/192897371-3c44b9b1-6aec-4e40-894b-6d111a26ad09.png)

Reply:
OK, sounds good